editoast/src/views/train_schedule/mod.rs

```diff
@@ -3,23 +3,19 @@ use std::collections::HashSet;
 use crate::core::{AsCoreRequest, CoreClient};
```

Comment:
There isn't a single existing test for this class, meaning no tests on the entirety of the train_schedule endpoints :( Not your fault though, since you're only slightly modifying existing code. Could you create an issue so they are hopefully added at some point ?

Reply:
https://github.com/osrd-project/osrd/issues/4632